### ISSUE TITLE
Add a reset button to MCM Pages

### ISSUE THREAD

**Issue by C3pa** [MWSE/MWSE#544]
It would be nice to add a "Restore Defaults" button to MCM Page, FilterPage, SideBarPage, and ExclusionsPage. That button would be behind an argument. On clicking the button, the each of the settings on the page that has a variable with `defaultSetting` field set, would get restored to that value. Extra care should be taken to make sure this would work with mwseMCMPlayerData variable since it's value isn't automatically initialized in the `player.data` in the MCM. 

Merlord's suggestion, placement-wise:
- Page - bottom righthand corner of the main window
- SidebarPage - bottom of right window
- FilterPage - bottom of right window
- ExclusionsPage - Bottom of the lefthand list

**Comment by C3pa** [comment]
Implementing this should be much easier after #546.

**Comment by herbertgithubaccount** [comment]
It might be nice to have a pop-up window appear asking for confirmation. e.g. "Are you sure you want to reset all the settings to their default values? This cannot be undone." But that might not be necessary.

**Comment by C3pa** [comment]
I agree, a confirmation popup sounds like a good idea.
